SharpGenTools.Sdk/SharpGenTools.Sdk.targets

```diff
@@ -261,24 +266,11 @@
     />
   </Target>
 
-  <Target
-    Name="GetGeneratedCSharpFiles"
-    DependsOnTargets="MapCppToCSharp"
-    Condition="'@(SharpGenMapping)' != ''"
-    Inputs="@(CSharpModel)"
-    Outputs="$(SharpGenIntermediateDir)ForceNotUpToDate">
-    <GetGeneratedCSharpFiles
-      Model="@(CSharpModel)"
-      GeneratedCodeFolder="$(SharpGenGeneratedCodeFolder)">
-      <Output TaskParameter="GeneratedFiles" ItemName="GeneratedCSharpFiles" />
-    </GetGeneratedCSharpFiles>
-  </Target>
-
   <Target
     Name="GenerateCSharp"
-    DependsOnTargets="CreateIntermediateDir;GetGeneratedCSharpFiles;RemoveGeneratedCSharpFiles"
+    DependsOnTargets="CreateIntermediateDir;MapCppToCSharp;RemoveGeneratedCSharpFiles"
     Inputs="@(SharpGenSdkAssembly);@(CSharpModel);@(DocLinksCache);@(SharpGenExternalDocs)"
-    Outputs="@(GeneratedCSharpFiles)"
+    Outputs="@(GeneratedCSharpFiles);$(SharpGenIntermediateDir)ForceNotUpToDate"
```

Comment:
By adding the `ForceNotUpToDate`, this is basically removing the point of the `Inputs` and `Outputs` fields.

Reply:
Indeed. That was a mistake made when removing `GetGeneratedCSharpFiles` target. `mine` branch does everything differently.